content/jamf/0001_connection_types/jamf.yaml

```
@@ -0,0 +1,30 @@
+# Code generated by openapi CLI. DO NOT EDIT.
+id: 41f64c5c-1f95-4901-b776-5ae32fcca0f5
+name: jamf
+plugin_name: http
+attributes:
+  PASSWORD:
+    display_name: Password
+    description: Your Jamf Password.
+    field_type: 0
+    input_type: password
+    placeholder: Password
+    required: true
+    index: 3
+  REQUEST_URL:
+    display_name: API Address
+    description: Jamf API address. The Jamf Pro API's base URL is located at /api
+      on yout Jamf Pro instance.
```

What does the thing about /api means?
I think it would be way clearer to give an example like this
```suggestion
    description: Jamf API address. Your API address would be https://instancename.jamfcloud.com if it's hosted in Jamf Cloud, or https://jamf.instancename.com:8443 if it's hosted on-premise.
```